### DISCUSSION TITLE
Differentiate showOutsideDays vs actual Days of the selected month

### DISCUSSION THREAD

**Discussion by hpdganesh** [gpbl/react-day-picker#1590] [Ideas]
When showOutsideDays is enabled, visually there is no difference between actual days of the selected month vs outside days. Can we add style to differentiate between actual days vs outside days

**Comment by gpbl** [comment]
@hpdganesh I am not sure I understand what do you mean. In DayPicker, you can change the style of the outside days with the `.rdp-day_outside`. See:

https://codesandbox.io/s/funny-carson-tq6jz4?file=/App.tsx

<img width="328" alt="Screenshot 2023-07-14 at 18 48 49" src="https://github.com/gpbl/react-day-picker/assets/120693/5b6b4b6c-5eab-4e34-a2d2-10211a638e88">
